airflow/models/trigger.py

```diff
@@ -218,10 +229,13 @@ def assign_unassigned(cls, triggerer_id, capacity, heartrate, session: Session =
         health_check_threshold = heartrate * 2.1
         alive_triggerer_ids = [
             row[0]
-            for row in session.query(Job.id).filter(
-                Job.end_date.is_(None),
-                Job.latest_heartbeat > timezone.utcnow() - datetime.timedelta(seconds=health_check_threshold),
-                Job.job_type == "TriggererJob",
+            for row in session.execute(
```

Comment:
Same as above, this can use `scalars` to get rid of the `row[0] for row in ...` thing